src/apprt/gtk/ui/menu-surface-context_menu.ui

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface domain="com.mitchellh.ghostty">
+  <requires lib="gtk" version="4.0"/>
+  <menu id="menu">
+    <section>
+      <item>
+        <attribute name="label" translatable="yes">Copy</attribute>
+        <attribute name="action">win.copy</attribute>
+      </item>
```

Comment:
Is this all hand-written? I feel like we might want some kind of mechanism to link this and the various actions in code together

Reply:
Yes, it's hand written. Yes, I think in a follow-up PR we'll want to convert it to a blueprint.